oom-frontend/src/app/core/auth/auth.service.ts

```diff
@@ -0,0 +1,45 @@
+import {BehaviorSubject, Observable, of} from 'rxjs';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533964)
```suggestion
import {BehaviorSubject, Observable, of} from "rxjs";
```